docs/acknowledgements.txt

```diff
@@ -92,6 +92,10 @@ support in NUT.*
 revived the 'sec' driver (as gamatronic), and expanded a bit genericups for its
 UPSs with alarm interface.
 
+* link:https://ever.eu/[EVER Power Systems], through David, added a
+USB HID subdriver for EVER UPSes (Sinline RT Series, Sinline RT XL Series,
+ECO PRO AVR CDS Series).
+
```

Comment:
Note @EVER-PowerSystems : I took the liberty to stub an acknowledgements entry here. Feel free to modify if I got some names or spellings wrong.

In particular, from the original PR comment I assume the developer responsible for this improvement was called David, but have no further details from the discussion or commits. A last name would be nice ;)

Reply:
David did not want to give his name. It can stay as you propose with the name only or simply as EVER Power Systems - this is our corporate account. Thanks

Reply:
Okay, thanks for the comment - I rephrased the acknowledgement to only name the company.